.gitignore

```diff
@@ -1,149 +1,274 @@
+!lib/common/win32/2015/release/cryptlib.lib
+!lib/common/win32/2015/release/d3dx9.lib
+!lib/common/win32/2015/release/libjpeg.lib
+!lib/common/win32/2015/release/libpng.lib
+!lib/common/win32/2015/release/libjpeg.lib
+!lib/common/win32/2015/release/libpng.lib
+!lib/common/win32/2015/release/d3dx9.lib
+!lib/common/win32/2015/release/binkw32.lib
+!lib/common/linux32/bzip2.a
+!lib/common/linux32/libSDL2_mixer.a
+!lib/common/linux32/libcrypto.a
+!lib/common/linux32/libcryptopp.a
+!lib/common/linux32/libcurl.a
+!lib/common/linux32/libcurlssl.a
+!lib/common/linux32/libjpeg.a
+!lib/common/linux32/libogg.a
+!lib/common/linux32/libspeex.a
+!lib/common/linux32/libssl.a
+!lib/common/linux32/libvorbis.a
+!lib/common/linux32/libvorbisenc.a
+!lib/common/linux32/libvpx.a
+!lib/common/linux32/libwebm.a
+!lib/common/osx32/bzip2.a
+!lib/common/osx32/libcrypto.a
+!lib/common/osx32/libjpeg.a
+!lib/common/osx32/libogg.a
+!lib/common/osx32/libspeex.a
+!lib/common/osx32/libvorbis.a
+!lib/common/osx32/libvorbisenc.a
+!lib/common/osx32/libvpx.a
+!lib/common/osx32/libwebm.a
+!lib/common/python/2.5/libpython25.a
+!lib/common/ubuntu12_32/libcrypto.a
+!lib/common/ubuntu12_32/libcryptopp.a
+!lib/common/ubuntu12_32/libssl.a
+!lib/common/ubuntu12_32_gcc48/libcrypto.a
+!lib/common/ubuntu12_32_gcc48/libcryptopp.a
+!lib/osx/libcelt.a
+!lib/osx32/libcelt.a
+!lib/osx32/replay_common.a
+!lib/public/linux32/appframework_srv.a
+!lib/public/linux32/bitmap_srv.a
+!lib/public/linux32/choreoobjects_srv.a
+!lib/public/linux32/dmxloader_srv.a
+!lib/public/linux32/gcsdk_srv.a
+!lib/public/linux32/havana_constraints_srv.a
+!lib/public/linux32/hk_base_srv.a
+!lib/public/linux32/hk_math_srv.a
+!lib/public/linux32/libpng.a
+!lib/public/linux32/libprotobuf-lite.a
+!lib/public/linux32/libprotobuf.a
+!lib/public/linux32/libprotoc.a
+!lib/public/linux32/libz.a
+!lib/public/linux32/lzma_srv.a
+!lib/public/linux32/mathlib_srv.a
+!lib/public/linux32/particles_srv.a
+!lib/public/linux32/replay_common_srv.a
+!lib/public/linux32/shaderlib_srv.a
+!lib/public/linux32/tier1_srv.a
+!lib/public/linux32/tier2_srv.a
+!lib/public/linux32/tier3_srv.a
+!lib/public/linux32/vtf_srv.a
+!lib/public/osx32/libcelt.a
+!lib/public/osx32/libpng.a
+!lib/public/osx32/libprotobuf-lite.a
+!lib/public/osx32/libprotobuf.a
+!lib/public/osx32/libprotoc.a
+!lib/common/osx32/libcurl.dylib
+!lib/public/osx32/libSDL2-2.0.0.dylib
+!lib/public/osx32/libSDL2.dylib
+!lib/public/osx32/libcef.dylib
+!lib/public/osx32/libopenvr_api.dylib
+!lib/public/osx32/libsteam_api.dylib
+!lib/public/osx32/libsteamnetworkingsockets.dylib
+!lib/public/linux32/libSDL2.so
+!lib/public/linux32/libcef.so
+!lib/public/linux32/libopenvr_api.so
+!lib/public/linux32/libsteam_api.so
+!lib/public/linux32/libsteamnetworkingsockets.so
+!dx9sdk/lib/d3dx9.lib
+!dx9sdk/lib/d3d9.lib
+!dx9sdk/lib/dinput8.lib
+!dx9sdk/lib/ddraw.lib
+!dx9sdk/lib/dsound.lib
+!dx9sdk/lib/dxguid.lib
+!tier0/DESKey/ALGO32.LIB
+!tier0/DESKey/DK2WIN.LIB
+.idea/
```

Comment:
Note to others: this is from changes in `master` that aren't upstreamed in `build-linux`